data/421/166/603/421166603.geojson

```diff
@@ -9,8 +9,11 @@
     "geom:bbox":"-84.05074,9.92829,-84.05074,9.92829",
     "geom:latitude":9.92829,
     "geom:longitude":-84.05074,
+    "gn:country":null,
```

Comment:
Probably don't want to store null key/value pairs?

Reply:
Just added a check for this one - I'll rerun with the new logic.